core/src/main/java/com/hcl/ecomm/core/services/impl/CustomerServiceImpl.java

```diff
@@ -71,6 +71,9 @@ public String customerProfileServicePath() {
         return config.customerService_profilePath();
     }
 
+    @Override
+    public String getRecaptchaSiteKey() { return  config.getRecaptchaSiteKey_string();}
```

Comment:
@nitheshkumar-hcl  Move this in common service impl

Reply:
Committed the changes as per the review comments